pkg/daemon/update.go

```diff
@@ -46,26 +43,11 @@ const (
 	// SSH Keys for user "core" will only be written at /home/core/.ssh
 	coreUserSSHPath = "/home/core/.ssh/"
 	// fipsFile is the file to check if FIPS is enabled
-	fipsFile = "/proc/sys/crypto/fips_enabled"
+	fipsFile          = "/proc/sys/crypto/fips_enabled"
+	extensionsRepo    = "/etc/yum.repos.d/coreos-extensions.repo"
+	osImageContentDir = "/run/mco-machine-os-content/"
```

Comment:
I don't see any code removing this directory when we're done with it.  I know right now we're rebooting, but extracting to `/run` is going to take up RAM until it's deleted.  Let's use an `ioutil.TempDir()` underneath there or so?

Reply:
yeah, we can do that. Reason of not deleting explicitly was we extract the OSContainer just before we start applying OS changes. Once OS changes get applied, we reboot the node.

Reply:
Pushed changes to extract content under a temp dir and delete them explicitly.